app/boards/shields/quefrency/quefrency_right.overlay

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2020 Noah Thornton
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include "quefrency.dtsi"
+
+&default_transform {
+    col-offset = <7>;
+};
+
+/ {
+
+    kscan0: kscan {
+        compatible = "zmk,kscan-gpio-matrix";
+        label = "KSCAN";
+        diode-direction = "col2row";
+
+
+
+        col-gpios
+            = <&pro_micro_a  2 GPIO_ACTIVE_HIGH>
+            , <&pro_micro_a  1 GPIO_ACTIVE_HIGH>
+            , <&pro_micro_a  0 GPIO_ACTIVE_HIGH>
+            , <&pro_micro_d 15 GPIO_ACTIVE_HIGH>
+            , <&pro_micro_d 14 GPIO_ACTIVE_HIGH>
+            , <&pro_micro_d 16 GPIO_ACTIVE_HIGH>
+            , <&pro_micro_d 10 GPIO_ACTIVE_HIGH>
+            , <&pro_micro_d  5 GPIO_ACTIVE_HIGH>
```

Comment:
I see this has one more column than the left side. Are these reversible? Truly have different number of columns? Or is this part of the "options" on 60% vs 65%

Reply:
They are not reversible. Each half has two PCB options. Left has a standard 60% and a 60%+macro pad. Right has a 60% and a 65%. I have the left 60% and right 65%, I am not sure how the matrix is on other combinations of this particular shield, but if someone has them it would be great to have all the possible layouts.

Reply:
Yeah, ok. Happy to get this in, and then we can update it once someone with the other hardware configurations needs it.

Can you add a comment to the left and right overlay files noting which specific option they are for?

Thanks.

Reply:
I added a bit about the possible PCB configurations, let me know if the format is good. I am not sure if there is some sort of existing convention in the ZMK code base. Let me know if there is anything else that needs tweaking.